core/tee/tee_svc.c

```diff
@@ -587,13 +587,25 @@ static TEE_Result utee_param_to_param(struct user_ta_ctx *utc,
 				      struct tee_ta_param *p,
 				      struct utee_params *up)
 {
+	TEE_Result res;
```

Comment:
This should be initialized according to our coding standards.

Reply:
ah I missed it. thanks!